packages/perseus/src/widgets/free-response/free-response.test.tsx

```diff
@@ -0,0 +1,87 @@
+import {screen} from "@testing-library/react";
+import {userEvent as userEventLib} from "@testing-library/user-event";
+
+import {testDependencies} from "../../../../../testing/test-dependencies";
+import * as Dependencies from "../../dependencies";
+import {renderQuestion} from "../__testutils__/renderQuestion";
+
+import {getFreeResponseItemData} from "./free-response.testdata";
+
+import type {APIOptions} from "../../types";
+import type {UserEvent} from "@testing-library/user-event";
+
+describe("free-response widget", () => {
+    let userEvent: UserEvent;
+
+    beforeEach(() => {
+        userEvent = userEventLib.setup({
+            advanceTimers: jest.advanceTimersByTime,
+        });
+
+        jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+            testDependencies,
+        );
+    });
+
+    it("should snapshot", () => {
+        // Arrange
+        const apiOptions: APIOptions = {
+            isMobile: false,
+        };
```

Comment:
I suspect this was done by following other tests. Ideally we rely less and less on the `isMobile` flag as that causes our code to diverge for mobile vs desktop. I think you could remove the test that checks for snapshotting on mobile and just remove the `isMobile` flag from APIOptions here. This widget, especially, feels like something that won't vary between desktop and mobile.. I could be wrong though.